### DISCUSSION TITLE
[Feature] Merge cells support

### DISCUSSION THREAD

**Discussion by hama0511** [iddan/react-spreadsheet#294] [Ideas]
Thanks for providing a good component.
Can you please support the function for merging cells specifically for columnLabels?

**Comment by iddan** [comment]
Hey, @hama0511 thank you for trying out React Spreadsheet and taking the time to suggest a feature!
I have limited time to develop new features for React Spreadsheet but this can definitely be on my backlog.
If you or someone reading this would like to develop this feature I would love to review PRs 🙏

**Comment by alexf101** [comment]
@iddan @hama0511 Hey, we might be interested in implementing this. Would you be able to provide any guidance on how you would like to see it done?

**Comment by iddan** [comment]
Hey @alexf101, sure.

**Comment by JurajKavka** [comment]
Hi, any plans to support merging the cells?
